src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ChangeLevelCalculator.kt

```diff
@@ -11,14 +11,35 @@ class ChangeLevelCalculator(
 ) {
 
   fun calculateChangeLevel(crn: String): TierLevel<ChangeLevel> {
-    val points = getOasysNeedsPoints(crn).plus(getOgrsPoints(crn))
-    val tier = when {
-      points >= 20 -> ChangeLevel.THREE
-      points in 10..19 -> ChangeLevel.TWO
-      else -> ChangeLevel.ONE
+    if (shouldCalculateChangeLevel(crn)) {
+
+      val points = getOasysNeedsPoints(crn).plus(getOgrsPoints(crn))
+      val tier = calculateTier(points)
+
+      return TierLevel(tier, points)
+    }
+    return TierLevel(ChangeLevel.ZERO, 0)
```

Comment:
Same here, have we calculated a tier level of 0 or have we not calculated a tier level.
I do have a recollection of siding on '0' for the int score here, but in context I wonder if the object should be null?

Reply:
I slacked you a conversation I had with Ed about this